en/people/floriana-ferrara.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Floriana Ferrara
+lang: en
+permalink: /en/people/floriana-ferrara.html
+layout: people
+role: Technical Project Manager
+is_new: true
+twitter_user: FlorianaFer
+medium_user: FlorianaFer
+linkedin_url: https://www.linkedin.com/in/filomena-floriana-ferrara-50588912 
+ref: floriana-ferrara
+parent_ref: team
+start_date: September 04, 2017
+period_provided: until September 16, 2018
+annual_compensation: € 80,000
+approved_by_court: September 24, 2017
+---
+I am from Taranto, I earned a degree in Information Science at University of Bari and I am a computer expert. 
+
+I spent two years at CORINTO (National Research Consortium for Objects Technology) in Bari and then I started working for IBM. 
+Here I have held various roles in different fields which are all related to the making of innovative digital projects.
+
+During a business meeting I met Diego and shortly after I became a member of the Digital Transformation Team, then I started the interview for the Technical Project Manager position. Being selected was a great honour and I am grateful to the IBM for having allowed me a year off to embark on this wonderful adventure.
```

Comment:
The english sentence here doesn't make much sense to me. "During a meeting .. I met $x, and SHORTLY AFTER I became, THEN I started the interview" (?)